lastuser_core/utils.py

```diff
@@ -41,6 +40,8 @@ def strip_phone(candidate):
 def valid_phone(candidate):
     return not PHONE_VALID_RE.search(candidate) is None
 
+def valid_username(candidate):
+    return not USERNAME_VALID_RE.search(candidate) is None
```

Comment:
The regex existed but there was no corresponding function to check for validity of username. So I added the function.

I just checked valid_username() is being used in lastuser is from coaster.utils. Nowhere else in code the regex is being used. So I guess there is no need for the regex?